code/modules/mining/ore.dm

```diff
@@ -16,6 +16,31 @@
 			user << "<span class='info'>Not enough fuel to smelt [src].</span>"
 	..()
 
+/obj/item/weapon/ore/Crossed(AM as mob|obj)
+	var/obj/item/weapon/storage/bag/ore/OB
+	var/turf/simulated/floor/F = get_turf(src)
+	if(loc != F)
+		return ..()
+	if(ishuman(AM))
+		var/mob/living/carbon/human/H = AM
```

Comment:
Well, rather, "ishuman" checks if the given thing is a "humanoid", though we call them human because humans were probably the only race who knows back when. Though generally, "human"s are the player mobs. I've tested this, worry not.